.gitignore

```diff
@@ -801,3 +801,9 @@ fabric.properties
 # Project
 /src/taxpasta/_version.py
 /.ruff_cache/
+taxpasta-tutorial/
+
+# Rstudio
+.Rhistory
+docs/taxpasta-tutorial/
+.Rproj.user
```

Comment:
I prefer we remove these entries before merging. If you want to ignore something that you're playing around with locally, it's better to put those entries into `.git/info/exclude`.

Reply:
Oh, I wasn't aware there was a different way to exclude files. What is the difference?

(Feel free to move yourself if you're making other changes)

Reply:
It won't be committed in the gitignore file but let's you ignore files for git locally anyway. So it's about sharing/not sharing the ignore settings.